service/db/dgclient.go

```diff
@@ -260,8 +273,51 @@ func (s DGClient) GetAllByClusterAndType(meta string, cluster string) (map[strin
 	return m, nil
 }
 
-//GetSchema - get all predicates
-func (s DGClient) GetSchema() ([]*api.SchemaNode, error) {
+//GetCacheContainsDBSchema - Get cache which contains db schema
+func (s DGClient) GetCacheContainsDBSchema() (*lru.Cache, error) {
+	//Add db schema to the cache
+	if InitLruCacheDBSchema == false {
```

Comment:
no need to compare boolean variable with false or true, simply write as
if !InitLruCacheDBSchema {
...
}

Reply:
Made the change.